### PR TITLE
change default filter to be based on previous two releases instead of a hardcoded date

### PR DESCRIPTION
**Pull request by bgrins**
Two functional changes here that we'll need to decide if we want:

1) This will get reset every time you reload the page (i.e. if you've selected all releases manually and then reload). This is an existing issue but currently it will reselect _all_ releases every time.
2) It deselects N/A which makes some of the reporting like "number of bugs by type" less useful (since it now doesn't include unfixed bugs). I wonder if we really should have a separate checkbox for "include unfixed bugs" rather than putting it into the release dropdown anyway.

### CODE REVIEW

**Comment by bgrins** [comment]
Do you think we should hold off on landing until we make a change to 1 or 2 above? I did 2 previous releases to avoid any confusion for the default filtering for QA teams (i.e. if all metabug work drops off right after a merge)

**Review by marco-c — APPROVED** [review]
> Do you think we should hold off on landing until we make a change to 1 or 2 above? I did 2 previous releases to avoid any confusion for the default filtering for QA teams (i.e. if all metabug work drops off right after a merge)

I'd say we can merge this and fix the caveats later.

**Comment by marco-c** [comment]
I filed #2079 and #2081 as follow-ups.

**Pull request merged**
